### DISCUSSION TITLE
Call a menu item from function

### DISCUSSION THREAD

**Discussion by janbelder** [forntoh/LcdMenu#160] [Q&A] [answered]
Hi Thomas,
Is it possible to go to a menu item directly?
I tried to figure that out but wasn't able to.
Hoped that I could find it in the link to gitbook somewhere here, but that link is broken....

**Comment by forntoh** [comment]
I don't really understand what you mean by going to a menu item directly, can you please explain more.

Also would be helpful if you paste the link which doesn't work here and where you got it from so that I can update it.

**Reply by rozrabiak** [reply]
Probably, push button 1 -> open menu and cursor is on position X
push button Z -> open menu and cursor is on position Y.

**Reply by forntoh** [reply, marked as answer]
You can use a combination of `menu.setCursorPosition‎(...)` and `menu.update()` to achieve what you want.
So when button Z is pressed the following code is executed

```cpp
menu.setCursorPosition‎(`your desired position goes here`)
menu.update() // redraw menu with changes
```
